babel.config.js

```diff
@@ -1,11 +1,3 @@
 const { createConfig } = require('@edx/frontend-build');
 
-module.exports = createConfig('babel-preserve-modules', {
-  env: {
-    test: {
-      plugins: [
-        'rewire',
```

Comment:
No longer needed. I made new httpClient exports from csrfToken and getJwtToken to allow for easier testing/mocking